helpers/jwt-service.go

```diff
@@ -0,0 +1,51 @@
+package helpers
+
+import (
+	"os"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/joho/godotenv"
+)
+
+type JWTService interface {
+	GenerateToken(user_id string) (string, error)
+	// ValidateToken(tokenString string) (*jwt.Token, error)
+}
+
+type JWTServiceInfo struct {
+	secretKey string
+	issuer    string
+}
+
+func NewJWTService() JWTService {
+	envError := godotenv.Load()
```

Comment:
I don't think you need to load it again since it is already loaded in the main func